README.md

```diff
@@ -0,0 +1,10 @@
+### Cafe
+
+The Beemo Cafe is a mono-repo containing all the public microservices that Beemo will, or is utilizing. In this mono-repo contains the following services:
+|                              Name                             |     Purpose     |                                     Description                                     |
+|:-------------------------------------------------------------:|:---------------:|:-----------------------------------------------------------------------------------:|
+|   [Latte](https://github.com/beemobot/cafe/tree/main/latte)   |     Commons     |  A set of Java-Kotlin commons that are used all over Beemo's Java-Kotlin services.  |
+|    [Milk](https://github.com/beemobot/cafe/tree/main/milk)    |   Public Logs   |            Handles storing and public viewing of logs such as raid logs.            |
```

Comment:
Since those are the only logs it stores under the new scope :D

```suggestion
|    [Milk](https://github.com/beemobot/cafe/tree/main/milk)    |   Public Logs   |            Handles storing and public viewing of raid logs.                         |
```